neo/Network/P2P/TaskManager.cs

```diff
@@ -221,7 +222,11 @@ private void RequestTasks(TaskSession session)
                     return;
                 }
             }
-            if ((!HasHeaderTask || globalTasks[HeaderTaskHash] < MaxConncurrentTasks) && Blockchain.Singleton.HeaderHeight < session.StartHeight)
+            if ((!HasHeaderTask || globalTasks[HeaderTaskHash] < MaxConncurrentTasks)
+                && (Blockchain.Singleton.HeaderHeight < session.StartHeight
```

Comment:
Please, put this information in variables. This is unreadable :(

Reply:
Outdated. Fixed.